src/alignment/configs.py

```diff
@@ -244,102 +245,15 @@ class SFTConfig(transformers.TrainingArguments):
     dataset_kwargs: Optional[Dict[str, Any]] = field(
         default=None, metadata={"help": "Dataset kwargs for the SFTTrainer"}
     )
-    max_seq_length: Optional[int] = field(
-        default=None,
-        metadata={"help": ("Used by TRL for reward model training, which tries to read this parameter in init.")},
-    )
-    logging_first_step: bool = field(
-        default=True,
-        metadata={"help": ("Whether to log and evaluate the first global_step or not.")},
-    )
-    optim: Optional[str] = field(default="adamw_torch")
 
 
 @dataclass
-class DPOConfig(transformers.TrainingArguments):
+class DPOConfig(trl.DPOConfig):
     """
     Arguments related to the DPO training process itself. For all parameters, see: https://huggingface.co/docs/transformers/v4.26.1/en/main_classes/trainer#transformers.TrainingArguments
     """
 
-    beta: Optional[float] = field(
-        default=0.1,
-        metadata={"help": "The beta factor in DPO loss. Higher beta means less divergence from the initial policy."},
-    )
     hub_model_revision: Optional[str] = field(
```

Comment:
Maybe not for this PR, but shouldn't we make sure that all the configs share `hub_model_revision`? As I believe it's useful for all of those when pushing models to the Hub right?

Reply:
added to SFT and DPO